### PR TITLE
Fix contradicting statements about invalid proof time values.

### PR DESCRIPTION
**Pull request by PatStLouis**
Addresses #294

***
<a href="https://pr-preview.s3.amazonaws.com/OpSecId/vc-data-integrity/pull/295.html" title="Last updated on Jul 28, 2024, 8:22 PM UTC (02e05c0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/295/a065cfc...OpSecId:02e05c0.html" title="Last updated on Jul 28, 2024, 8:22 PM UTC (02e05c0)">Diff</a>

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

**Review by Wind4Greg — APPROVED** [review]

**Review by BigBlueHat — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -773,17 +773,15 @@ <h3>Proofs</h3>
 The date and time the proof was created is OPTIONAL and, if included, MUST be
 specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string, either in Universal
 Coordinated Time (UTC), denoted by a Z at the end of the value, or with a time
-zone offset relative to UTC. Time values that are incorrectly serialized without
-an offset MUST be interpreted as UTC.
+zone offset relative to UTC.
```
Comment (dlongley) [inline index.html]:
```suggestion
zone offset relative to UTC. If time values that were incorrectly serialized
without an offset are accepted by a consumer, they are to be interpreted
as UTC.
```

File: index.html [hunk 2]
```diff
@@ -773,17 +773,15 @@ <h3>Proofs</h3>
...
           </dd>
 
           <dt id="defn-proof-expires">expires</dt>
           <dd>
 The `expires` property is OPTIONAL and, if present, specifies when the proof
 expires. If present, it MUST be an [[XMLSCHEMA11-2]] `dateTimeStamp` string,
 either in Universal Coordinated Time (UTC), denoted by a Z at the end of the
-value, or with a time zone offset relative to UTC. Time values that are
-incorrectly serialized without an offset MUST be interpreted as UTC.
+value, or with a time zone offset relative to UTC.
```
Comment (dlongley) [inline index.html]:
```suggestion
value, or with a time zone offset relative to UTC. If time values that
were incorrectly serialized without an offset are accepted by a
consumer, they are to be interpreted as UTC.
```

**Review by dlongley — COMMENTED** [review]
Providing suggestions from the related issue.

**Review by msporny — APPROVED** [review]
With @dlongley suggestions included, LGTM.

**Comment by PatStLouis** [comment]
@dlongley thank you for the suggestion. I also approve this change.

**Review by dlongley — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -773,17 +773,19 @@ <h3>Proofs</h3>
 The date and time the proof was created is OPTIONAL and, if included, MUST be
 specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string, either in Universal
 Coordinated Time (UTC), denoted by a Z at the end of the value, or with a time
-zone offset relative to UTC. Time values that are incorrectly serialized without
-an offset MUST be interpreted as UTC.
+zone offset relative to UTC. If time values that were incorrectly serialized
+without an offset are accepted by a consumer, they are to be interpreted
+as UTC.
```
Comment (PatStLouis) [inline index.html]:
```suggestion
zone offset relative to UTC. A [=conforming processor=] MAY chose to consume
time values that were incorrectly serialized without an offset. Incorrectly
serialized time values without an offset are to be interpreted as UTC.
```

File: index.html [hunk 4]
```diff
@@ -773,17 +773,19 @@ <h3>Proofs</h3>
...
           <dt id="defn-proof-expires">expires</dt>
           <dd>
 The `expires` property is OPTIONAL and, if present, specifies when the proof
 expires. If present, it MUST be an [[XMLSCHEMA11-2]] `dateTimeStamp` string,
 either in Universal Coordinated Time (UTC), denoted by a Z at the end of the
-value, or with a time zone offset relative to UTC. Time values that are
-incorrectly serialized without an offset MUST be interpreted as UTC.
+value, or with a time zone offset relative to UTC. If time values that
+were incorrectly serialized without an offset are accepted by a
+consumer, they are to be interpreted as UTC.
```
Comment (PatStLouis) [inline index.html]:
```suggestion
value, or with a time zone offset relative to UTC. A [=conforming processor=]
MAY chose to consume time values that were incorrectly serialized without an
offset. Incorrectly serialized time values without an offset are to be
interpreted as UTC.
```

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**